content/en/docs/how-tos/cluster-claim.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Creating a Cluster Pool"
+description: How to create a cluster pool from which a job can claim a cluster.
+---
+
+DPTP integrates Hive into the OpenShift test platform to manage the cluster pools and clusters in them. There is a defined pool [ci-cluster-pool](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) from which can be used to claim an OpenShift cluster for a test. In this page, we document how to create a new cluster pool with the cloud account provided by a CI user.
+
+## Prerequisites
+
+*  Choose a [Hive](https://github.com/openshift/hive#supported-cloud-providers) supported cloud for the pool.
+* [Configure the cloud account](https://docs.openshift.com/container-platform/4.7/installing/installing-preparing.html).
+
+## Create a cluster pool
+
+Create a folder in `release/clusters/app.ci/` with the manifests for the pool. See [the examples](https://github.com/openshift/release/tree/master/clusters/app.ci/ci-cluster-pool) for the existing `ci-cluster-pool`.
+
+```yaml
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  name: ci-ocp-4-7-amd64-aws-us-east-1
+  namespace: cvp-cluster-pool
+  labels: # product, version, architecture, cloud, owner are used to filter out a pool when a job claims a cluster
+    product: ocp
+    version: "4.7"
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    region: us-east-1
+spec:
+  baseDomain: hive.example.org # the base domain to install the cluster
+  imageSetRef:
+    name: ocp-4.7.0-amd64 # the name of the imageSet which determines the image to install the cluster
+  installConfigSecretTemplateRef:
+    name: install-config-aws-us-east-1 # the name of the secret with an installation config for the installer
+  skipMachinePools: true
+  platform:
+    aws:
+      credentialsSecretRef:
+        name: hive-aws-credentials # the name of the secret with the credentials of the cloud account
+      region: us-east-1
+  pullSecretRef:
+    name: pull-secret # the name of the pull secret used by installer
+  size: 1
+  maxSize: 10
+```
+
+The referred manifests in the pool are in the same folder. The meaning of other fields in the pool specification is available at [Hive's documentation](https://pkg.go.dev/github.com/openshift/hive/apis@master/hive/v1#ClusterPool).
```

Comment:
We should IMO at least briefly discuss the sizing and resource consumption.

Reply:
Updated.